### DISCUSSION TITLE
How to open in VS?

### DISCUSSION THREAD

**Discussion by mcrdotnetcore** [dotnet/AspNetCore.Docs#24803] [General]
Sorry this is a stupid question, but I am only 6 months in.

How do I open these examples in Visual Studio? I am finding myself looking for a solution to open but can't find any?

Appreciate the warm responses! 
Thanks

**Comment by Rick-Anderson** [comment]
What samples? Solutions aren't stored in GitHub for most samples because the solution file is not needed. Open the project file.
